modules/ROOT/pages/managing-users.adoc

```diff
@@ -50,6 +50,16 @@ Public key provided by the identity provider, which is used to sign the SAML ass
 +
 An arbitrary string value that identifies your Anypoint Platform organization. The typical value for this string is `<organizationDomain>.anypoint.mulesoft.com`.
 +
+* *Single Sign-On Initiation*
```

Comment:
Why is this bullet item boldface and in title style?
Is this a UI label?
Otherwise
```suggestion
* Single sign-on initiation*
```
And we are using "single sign-on" because of SSO rather than our verb of "sign in to"?

Reply:
It is a UI label, yes.